tools/common/Frameworks.cs

```diff
@@ -629,7 +629,6 @@ public static Frameworks GetMacCatalystFrameworks ()
 				case "HealthKitUI":
 				case "iAd":
 				case "IdentityLookupUI":
-				case "HomeKit":
```

Comment:
I said this could wait, but if you want to be adventurous then I'm fine with that too :smile:

Reply:
I can remove it! I just wanted to see if it caused more issues! haha